semgrep-core/parsing/Parse_kotlin_tree_sitter.ml

```diff
@@ -516,28 +517,29 @@ and binary_expression (env : env) (x : CST.binary_expression) =
         | `Is_op x -> is_operator env x
         )
       in
+      let v2_id = Id (("", v2), empty_id_info()) in
```

Comment:
We could add an operator for that maybe. It's actually used also in Python (but right now I use
OE_In, and OE_NotIn, but really we could make In and NotIn operators like Plus/... in AST_generic.operator.

Reply:
adding in AST_generic sounds great, we could probably also add in the Is/NotIs operators in there too 🤔